debian/rules

```diff
@@ -9,23 +9,44 @@
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
+DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+
 %:
-	dh $@ 
+	@for target in `echo $(shell find debian/*.install.in)`; do \
```

Comment:
Where the install targets with the asterisks broken, that this is needed?

Reply:
The standard *.install with:
```
usr/lib/*
usr/share/*
```
does not work on this addon with several addons in one and bring on every addon there the complete content of all (e.g. kodi-screensaver-rsxs-biof), see content: https://pastebin.com/98GQeSpH on a at 23 packages with 77 MB each.

The `*.install.in` with e.g.:
```
usr/lib/@DEB_HOST_MULTIARCH@/kodi/addons/screensaver.rsxs.helios/* /usr/lib/@DEB_HOST_MULTIARCH@/kodi/addons/screensaver.rsxs.helios
usr/share/kodi/addons/screensaver.rsxs.helios/*
```
prevent this and include only the required parts.